CP04OSSM/CP04OSSM_R00014_ingest.csv

```diff
@@ -0,0 +1,42 @@
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/cpm3/syslog/cpm_status.txt,CP04OSSM-MFC31-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/syslog/*.syslog.log,CP04OSSM-MFD35-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/adcp/*adcp*.log,CP04OSSM-MFD35-01-ADCPSJ000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/instruments/dcl36/ADCP*/*RDI*.000,CP04OSSM-MFD35-01-ADCPSJ000,recovered_inst,Available,
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/presf/*.presf.log,CP04OSSM-MFD35-02-PRESFC000,recovered_host,Available,
+mi.dataset.driver.presf_abc.presf_abc_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/instruments/dcl36/PRESF*/PRESF*.hex,CP04OSSM-MFD35-02-PRESFC000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/velpt2/*velpt*.log,CP04OSSM-MFD35-04-VELPTB000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/instruments/dcl36/VELPT*/*.aqd,CP04OSSM-MFD35-04-VELPTB000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/pco2w/*.pco2w.log,CP04OSSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/instruments/dcl36/PCO2W*/SAMI*.txt,CP04OSSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl36/phsen2/*.phsen2.log,CP04OSSM-MFD35-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/instruments/dcl36/PHSEN*/SAMI*.txt,CP04OSSM-MFD35-06-PHSEND000,recovered_inst,Available,
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl37/syslog/*.syslog.log,CP04OSSM-MFD37-00-DCLENG000,recovered_host,Not Expected,DCL37 damaged prior to deployment; no data expected
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl37/optaa2/*.optaa2.log,CP04OSSM-MFD37-01-OPTAAD000,recovered_host,Not Expected,Instrument removed prior to deployment; no data expected
+#mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00014/cg_data/dcl37/ctdbp2/*ctdbp2.log,CP04OSSM-MFD37-03-CTDBPE000,recovered_host,Not Expected,Instrument removed prior to deployment; no data expected

```

Comment:
the only instrument removed was the OPTAA I think?  So not sure why the CTDBP has note that it was removed prior to deployment and is commented out?